crates/relayer-types/src/timestamp.rs

```diff
@@ -153,6 +153,11 @@ impl Timestamp {
             _ => false,
         }
     }
+
+    /// Yields the Timestamp as a string.
+    pub fn as_str(&self) -> String {
+        format!("{}", self.nanoseconds())
+    }
```

Comment:
I would rather we call `timestamp.nanoseconds().to_string()` where needed rather than add a bespoke `to_str` method whose output does not match with the `Display` instance below.

Alternatively we can rename this method to `to_nanoseconds_string` but that seems a bit too ad-hoc.

Reply:
Also removed the method and used `.nanoseconds().to_string()` instead.